src/CMakeLists.txt

```diff
@@ -256,6 +258,10 @@ set( MIOpen_Source
     solver/conv_winoRxS.cpp
     solver/conv_winoRxS_fused.cpp
     solver/fft.cpp
+    solver/fold/unfold_forward.cpp
+    solver/fold/unfold_backward.cpp
+    solver/fold/fold_forward.cpp
+    solver/fold/fold_backward.cpp
```

Comment:
plz sort

Reply:
resolve in [e76095e](https://github.com/moreh-dev/MIOpen/pull/37/commits/e76095e8fc1771a82978af13018b538b396877d1)